core/scripts/local/RequestOraclePrice.js

```diff
@@ -50,31 +59,15 @@ async function run(registryAddress, oracleAddress, identifier, timeInSeconds) {
 }
 
 const runRequestOraclePrice = async function(callback) {
-  // Usage: truffle exec scripts/RequestOraclePrice.js <Registry address> <CentralizedOracle address> <identifier> <time> --network <network>
+  // Usage: truffle exec scripts/RequestOraclePrice.js <identifier> <time> --network <network>
   // where <time> is seconds since January 1st, 1970 00:00:00 UTC.
-  if (process.argv.length < 8) {
-    console.error(
-      "Not enough arguments. Must include <Registry address>, <CentralizedOracle address>, <identifier>, and <time>"
-    );
+  if (process.argv.length < 4) {
```

Comment:
This seems a little hacky. I know this is just pulled over from `v0`, but while we're modifying it, should we check the individual arguments instead of just checking the length? Something like:
```
if (!argv.time) {
...
return;
}
if (!argv.identifier) {
...
return;
}
```

Reply:
Done.